### PR TITLE
Seperated search functionality

### PR DESCRIPTION
**Pull request by exeokan**
Seperated the search for function, printing line by line in main.py

### CODE REVIEW

**Comment by The-B0nsai** [comment]
deneme

**Comment by recepsirin0** [comment]
Function name can be better.

File: main.py
Comment (exeokan) [inline main.py:1]: function name could be batter

File: search_function.py
Comment (exeokan) [inline search_function.py:1]: also change here

**Review by exeokan — COMMENTED** [review]
please consider

File: main.py [hunk 1]
```diff
@@ -1,11 +1,9 @@
+from search_function import search_lines
 def main():
     with open('input.txt', 'r') as file:
         search_word = input('Enter the search word: ')
         lines = file.readlines()
-        for line in lines:
-            if line.find(search_word) == -1:
-                continue
-            print(line.strip())
+        search_lines(lines, search_word)
```
Comment (exeokan) [inline main.py:6]: this line exactly, change pls

**Pull request merged**